src/AutosuggestContainer.js

```diff
@@ -126,29 +128,30 @@ export default class AutosuggestContainer extends Component {
     } = this.props;
 
     return (
-      <Autosuggest
-        customInput={customInput}
-        multiSection={multiSection}
-        shouldRenderSuggestions={shouldRenderSuggestions}
-        suggestions={suggestions}
-        onSuggestionsUpdateRequested={onSuggestionsUpdateRequested}
-        getSuggestionValue={getSuggestionValue}
-        renderSuggestion={renderSuggestion}
-        renderSectionTitle={renderSectionTitle}
-        getSectionSuggestions={getSectionSuggestions}
-        inputProps={inputProps}
-        onSuggestionSelected={onSuggestionSelected}
-        focusInputOnSuggestionClick={focusInputOnSuggestionClick}
-        selectFirst={selectFirst}
-        auxiliarComponent={auxiliarComponent}
-        auxiliarComponentPosition={auxiliarComponentPosition}
-        theme={mapToAutowhateverTheme(theme)}
-        id={id}
-        inputRef={this.saveInput}
-        store={this.store}
-        customRenderInput={customRenderInput}
-        customRenderList={customRenderList}
-      />
+      <Provider store={this.store}>
```

Comment:
Remember..  This store will be the store for all components down in tree. So, `AddressTypeahead` and other components will have this store as main one. Just a friendly reminder <3